### PR TITLE
Fix converter station modification issue in unbuilt state

### PR DESCRIPTION
**Pull request by jamal-khey**
This pull request fixes an issue with modifying converter stations in the network. Previously, the modification process was retrieving the converter stations from user inputs leading to errors.  
this fix retrieve the converter stations from the network wich is more reliable 

see related pull request [gridstudy-app](https://github.com/gridsuite/gridstudy-app/pull/2090)

### CODE REVIEW

File: src/test/java/org/gridsuite/modification/server/modifications/VscModificationTest.java [hunk 1]
```diff
@@ -262,6 +262,21 @@ public void testUnchangedHVDCangleDroopActivePowerControl() throws Exception {
         Assert.assertTrue(activePowerControl.isEnabled());
     }
 
+    //Test : p0 sould be required if drop is enabled
+    @Test
+    public void testHvdcAngleDroopActivePowerControlWithoutP0() {
```
Comment (AAJELLAL) [inline src/test/java/org/gridsuite/modification/server/modifications/VscModificationTest.java:267]: I think you should add one more test to cover this the scenario:
- p0 sould not be required if drop is disabled.

File: src/main/java/org/gridsuite/modification/server/modifications/VscModification.java [hunk 2]
```diff
@@ -51,10 +51,22 @@ public void check(Network network) throws NetworkModificationException {
...
+        VscConverterStation converterStation1 = ModificationUtils.getInstance().getVscConverterStation(network, hvdcLine.getConverterStation1().getId());
+        VscConverterStation converterStation2 = ModificationUtils.getInstance().getVscConverterStation(network, hvdcLine.getConverterStation2().getId());
         checkConverterStation(modificationInfos.getConverterStation1(), converterStation1);
         checkConverterStation(modificationInfos.getConverterStation2(), converterStation2);
+
+        checkToModifyDroop();
+
+    }
+
+    private void checkToModifyDroop() {
```
Comment (AAJELLAL) [inline src/main/java/org/gridsuite/modification/server/modifications/VscModification.java:65]: It's my opinion but I don't find the name appropriate.
Suggestion: checkDroopModification, isDropModificationAllowed...

File: src/test/java/org/gridsuite/modification/server/modifications/VscModificationTest.java (hunk 1 above)
Comment (jamal-khey) [reply src/test/java/org/gridsuite/modification/server/modifications/VscModificationTest.java:266]: done

File: src/main/java/org/gridsuite/modification/server/modifications/VscModification.java (hunk 2 above)
Comment (jamal-khey) [reply src/main/java/org/gridsuite/modification/server/modifications/VscModification.java]: done with isDropModificationAllowed

**Review by AAJELLAL — APPROVED** [review]
Test ans code OK.

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**